BootloaderCorePkg/Tools/PrepareBuildComponentBin.py

```diff
@@ -52,8 +56,9 @@ def CheckFileListExist (copy_list, sbl_dir):
     return exists
 
 
-def CopyFileList (copy_list, src_dir, sbl_dir):
+def CopyFileList (copy_list, src_dir):
```

Comment:
let us keep this function more generic , and do not depends on env variables.

Reply:
OK. will add it back.